autogluon/utils/tabular/ml/models/catboost/catboost_model.py

```diff
@@ -201,3 +201,10 @@ def fit(self, X_train, Y_train, X_test=None, Y_test=None, time_limit=None, **kwa
                 self.model.shrink(ntree_start=0, ntree_end=best_iteration+1)
 
         self.params_trained['iterations'] = self.model.tree_count_
+
+    def get_model_feature_importance(self):
```

Comment:
Just want to confirm these importances will be on similar scale to those computed by the other models using `AbstractModel. compute_permutation_importance()`?

Reply:
They are not, this is currently used to get features which are never used by the models (such as never used in a split). It is impossible to scale them to be identical to permutation importance unfortunately, since it varies by model.